code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm

```diff
@@ -1,4 +1,4 @@
-/datum/anvil_recipe/valuables
+	/datum/anvil_recipe/valuables
```

Comment:
Strange indentation, don't know if it will work as is but its better to set it right